api/product-sitemap.js

```
@@ -0,0 +1,66 @@
+const SITE_URL = "https://www.supermerch.com.au";
+const BACKEND_URL = process.env.BACKEND_URL || process.env.VITE_BACKEND_URL || "https://api.supermerch.com.au";
```

ESLint

[error] 'process' is not defined.

(no-undef)